unity3d/Assets/Swrve/SwrveSDK/SwrveSDKImp.cs

```diff
@@ -89,8 +89,8 @@ public partial class SwrveSDK
     protected bool campaignsAndResourcesInitialized;
 
     // Talk related
-    private static readonly int CampaignAPIVersion = 1;
-    private static readonly int CampaignEndpointVersion = 4;
+    private static readonly int CampaignEndpointVersion = 6;
```

Comment:
@adam-govan here.  Here is where I do that thing you mentioned.  Riiiiight here.